tools/docker/tests/test_flows.sh

```diff
@@ -155,8 +155,10 @@ test_client_association() {
 test_higher_layer_data_payload_trigger() {
     status "test higher layer data payload"
     
-    mac_gateway_hex=`echo $mac_gateway | tr --delete :`
-    mac_gateway_hex="0x${mac_gateway_hex}"
+    mac=
```

Comment:
Why is there a new line after assignment?
This should be
`mac_gateway=$(mac_to_hex $mac_gateway)`